scripts/mkcloud

```diff
@@ -950,6 +950,8 @@ function cloudupgrade()
 {
     if iscloudver 5plus ; then
         crowbarupgrade_5plus
+        onadmin crowbar_shutdown_services
+        onadmin crowbar_openstack_backup
```

Comment:
I agree that needs some clarification. IMHO if the function calls belong to `crowbarupgrade_5plus()` then why isn't the call of `onadmin proposal` there as well? I see it as part of the upgrade as well.